api/API_MANIFESTO.adoc

```diff
@@ -12,20 +12,20 @@ Examples:
  
  * Tasks, concurrency, counters uses cases
 
-The dependencies to thirty-party libraries must be avoided. This module should contain dependencies to JDK classes, 
-not more.
+The dependencies to thirty-party libraries must be avoided. This module should contain dependencies to JDK classes, not more.
 
 If you really **need** for a reason to use an external library to be part of an API. 
 Here is a non-exhaustive list of what we should discuss before promoting an API that uses an external dependency:
 
--Is there a way to keep that library in implementation instead of public API? Creating our Infinispan 
+- Is there a way to keep that library in implementation instead of public API? Creating our Infinispan 
```

Comment:
"Is it possible to keep the library as an implementation detail instead of promoting it to the API module?"

Reply:
Done